### PR TITLE
MATLAB 2020a

### PR DESCRIPTION
**Pull request by orbsmiv**
For INC1026810

Requires: https://github.com/bear-rsg/easybuild-easyblocks/pull/26

Requires: https://gitlab.bham.ac.uk/res-comp-team/system-modules/-/merge_requests/26

`MATLAB-2020a.eb`
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/m/MATLAB/MATLAB-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+java_options = '-Xmx2048m'
+
+# We encode a dummy licence server host/port and then set the MLM_LICENSE_FILE environment
+# variable in the licences module
+license_server = 'dummy.local'
+license_server_port = '1234'
+# key = '00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000'
+
+# For installation on BlueBEAR set the EB_MATLAB_KEY env var with a comma-separated list of installation keys,
+# e.g. MATLAB key, Parallel Server key etc.
```
Comment (edmondac) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-2020a.eb:28]: We've removed that EB_MATLAB_KEY env var from eb-conf... So I presume this whole commented block can be removed now?

Comment (orbsmiv) [reply easybuild/easyconfigs/m/MATLAB/MATLAB-2020a.eb:28]: Added further info to comments in 8f68a54

**Comment by edmondac** [comment]
Building now. The instructions for getting the key worked :-)

**Comment by orbsmiv** [comment]
@edmondac I've updated the key instructions at https://bearwiki.bham.ac.uk/bear/doku.php?id=arc:rsg:bear-apps:building:app-specific:matlab, can you build again to include _MATLAB Parallel Server_?

**Pull request merged**